tests/test_glacier.py

```diff
@@ -0,0 +1,137 @@
+from common import BaseTest
+from c7n.executor import MainThreadExecutor
+from c7n.resources.glacier import Glacier
+
+
+class GlacierTagTest(BaseTest):
+
+    def test_glacier_tag(self):
```

Comment:
given this is self contained it can use the functional decorator in tests/common.py